src/backgrounds.jl

```diff
@@ -0,0 +1,212 @@
+"""
+
+`draw_grid(video::Video, action::Action, frame::Int; grid_direction::Symbol = :TR)`
+
+Draws an oriented grid on the given frame of a Video.
+
+# Arguments
+- `grid_direction::Symbol`: Where grid animation finishes. Default: `:TR` Available Orientations:
+  - `:TR` - Animation finishes in the **T**op **R**ight corner of the frame.
+  - `:TL` - Animation finishes in the **T**op **L**eft corner of the frame.
+  - `:BR` - Animation finishes in the **B**ottom **R**ight corner of the frame.
+  - `:BL` - Animation finishes in the **B**ottom **L**eft corner of the frame.
+
+# Example
+This example will produce a grid animation that finishes in the top left corner of the animation. One will need to define their own path for `tempdirectory` and `pathname`.
+
+```
+using Javis
+using Luxor
+
+function ground(args...)
+    background("white")
+    sethue("black")
+end
+
+video = Video(500, 500)
+javis(
+      video,
+      [
+          Action(1:100, ground),
+          Action(1:100, (args...) -> draw_grid(args..., grid_direction = :TL)),
```

Comment:
The user would prefer to write `draw_grid(direction=:TL)` I assume. So using a `close` is probably the way to go.
Maybe have a look at #15 
(I would use `direction` as it's inside the `grid` function already.)

Reply:
More concretely: https://github.com/Wikunia/Javis.jl/blob/7b63527d9b56c463bc74c8d0636d6cbaeb5a754f/src/Javis.jl#L323

Reply:
My first closure! Always wanted to use one of these things. :smile: Done!